### ISSUE TITLE
SessionData.ProfileData needs a null check

### ISSUE THREAD

**Issue by deeja** [lowedown/xConnectHelper#2]
If there is no session, then ProfileData is null and a Null Reference occurs
```
  <% if (SessionData.ProfileData.Any()) { %>
                    <tr>
                        <td>Profiles:</td>
						<td>
                            <ul>
                            <% foreach (var profile in SessionData.ProfileData)
                                { %>
                                <li><%= profile %></li>
                            <%} %>
                            </ul>
						</td>
                    </tr>
                    <% } %>
```

**Comment by deeja** [comment]
Thanks for this tool by the way!

**Comment by lowedown** [comment]
Thanks for reporting. This is fixed now in V1.0
